src/javascript/js-webshim/minified/polyfiller.init.js

```diff
@@ -0,0 +1,6 @@
+(function($) {
+    $.webshim.bugs.bustedValidity = !('minLength' in document.createElement('input'));
+    $.webshims.setOptions('basePath', Zikula.Config.baseURL+'javascript/js-webshim/minified/shims/'); 
+    $.webshims.activeLang(Zikula.Config.lang);
+    $.webshims.polyfill(Zikula.Config.polyfillFeatures);
+})(jQuery);
```

Comment:
this version doesn't appear to be 'minified' in anyway - do we really need to two copies of the same source? how about just one?